src/integrationtest/java/com.microsoft.aad.msal4j/RefreshTokenIT.java

```diff
@@ -72,13 +72,12 @@ public void acquireTokenWithRefreshToken() throws Exception{
 
         Assert.assertNotNull(result);
         Assert.assertNotNull(result.accessToken());
-        Assert.assertNotNull(result.refreshToken());
```

Comment:
>Assert.assertNotNull(result.refreshToken()); [](start = 8, length = 44)

Why?

Reply:
we do not expose RT through IAuthenticationResult